web/src/engine/websites/decorators/Liliana.ts

```diff
@@ -0,0 +1,42 @@
+//For websites similar to MangaSect, Manga347, ManhuaPlus, ManhuaGold,  MangaReader.to,
+
+export const mangaPath ='/all-manga/{page}/';
+export const queryMangaTitleFromURI = 'div.anisc-detail .manga-name';
+export const queryMangas = 'div.item div.manga-detail .manga-name a';
+export const queryChapters = [
+    'ul#chapters-list li a.item-link',
+    'ul#myUL li.chapter a',
+].join(',');
+
+export const queryPagesScript = `
+    new Promise(resolve => {
+
+        function parseResults(data) {
+            const dom = new DOMParser().parseFromString(data, 'text/html');
+            let nodes = [...dom.querySelectorAll('div.separator')];
+
+            //sort if needed
+            if (nodes[0].hasAttribute('data-index')) {
```

Comment:
What about the case when some nodes have this attribute, and some nodes don't?

Reply:
This case was never encountered. I mean, the website script itself is reordering them. In that case it would be broken.
Its all or nothing.

Reply:
Handled in sorting anyway.